packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -824,16 +827,17 @@ async function processRequest(params) {
   if (reportType === 'Granule Inventory') reportKey = reportKey.replace('.json', '.csv');
 
   // add request to database
-  const reconciliationReportModel = new ReconciliationReport();
-  const reportRecord = {
+  const reconciliationReportPgModel = new ReconciliationReportPgModel();
+  const builtReportRecord = {
     name: reportRecordName,
     type: reportType,
     status: 'Pending',
     location: buildS3Uri(systemBucket, reportKey),
   };
-  let apiRecord = await reconciliationReportModel.create(reportRecord);
-  await indexReconciliationReport(esClient, apiRecord, process.env.ES_INDEX);
-  log.info(`Report added to database as pending: ${JSON.stringify(apiRecord)}.`);
+  let [reportPgRecord] = await reconciliationReportPgModel.create(knex, builtReportRecord);
+  let reportApiRecord = translatePostgresReconciliationReportToApiReconciliationReport(reportPgRecord); // eslint-disable-line max-len
```

Comment:
Yeah it does get around the max length. I found it was used in other places and had to use it in a few cases where I didn't think there was a better way of shortening the lines. I definitely think it should be used sparingly, so in any of the places I've used it, if there are good ways of shortening those lines I'm very open to suggestions.

Reply:
Yup, it's not the end of the world but preference is definitely to make lint happy. Separating arguments on separate lines is great but I do see where you'll have trouble with the very long function name. Could do something like "Recon" instead of "Reconciliation" if we're really gonna nit pick 😄